contents/handbook/company/communication.md

```diff
@@ -104,6 +104,16 @@ GitHub Issues are useful when there isn't a specific code change that is being p
 
 However, it is still important to maintain focus when opening issues by defining a single specific topic of discussion as well as defining the desired outcome that would result in the resolution of the issue. The point is to not keep issues open-ended and to prevent issues from going stale due to lack of resolution. For example, a team member may open an issue to track the progress of a blog post with associated to-do items that need to be completed by a certain date (e.g. first draft, peer review, publish). Once the specific items are completed, the issue can successfully be closed.
 
+### Product definitions & decisions
```

Comment:
I see a couple of benefits ([a] forces a conclusion, final documented decision; [b] it's a bit easier to do threaded discussions; [c] makes it clear who's expected to provide feedback) to this approach but I'm **not sure if this will solve the underlying problem**. It's just as easy for things to extend out of control in a PR as in an issue.

We also have discussions where the discussion gets quite long, yet it's not stale (a ton of value is still being created), but it does get to the same point where it becomes impossible for someone out of the loop to gather all context. Not sure if we can propose a better approach to this. Examples: https://github.com/PostHog/posthog/issues/5665; https://github.com/PostHog/posthog/issues/5543.